src/Symfony/Component/HttpKernel/DataCollector/LoggerDataCollector.php

```diff
@@ -47,7 +51,11 @@ public function lateCollect()
     {
         if (null !== $this->logger) {
             $this->data = $this->computeErrorsCount();
-            $this->data['logs'] = $this->sanitizeLogs($this->logger->getLogs());
+
+            $containerDeprecationLogs = $this->getContainerDeprecationLogs();
+            $this->data['deprecation_count'] += count($containerDeprecationLogs);
+            $this->data['container_compiler_logs'] = $this->getContainerCompilerLogs();
```

Comment:
let's rename this to compiler_logs to be consistent with the public method `getCompilerLogs`